### ISSUE TITLE
Add custom critical icon to a PF icons set

### ISSUE THREAD

**Issue by doruskova** [patternfly/patternfly#4747]
Regarding this issue https://github.com/patternfly/patternfly-design/issues/1138
I created an icon that presents a critical total risk value for Insights Advisor. 

- SVG file here [critical-risk.svg.zip](https://github.com/patternfly/patternfly/files/8331998/critical-risk.svg.zip)
- Updated Icon exporter [icon-exporter.sketch.zip](https://github.com/patternfly/patternfly/files/8332007/icon-exporter.sketch.zip)
- AI file with an icon [critical-risk-icon.ai.zip](https://github.com/patternfly/patternfly/files/8332012/critical-risk-icon.ai.zip)
<img width="174" alt="Snímek obrazovky 2022-03-23 v 11 14 28" src="https://user-images.githubusercontent.com/32618096/159676132-71068e3d-67bd-4bda-ab4d-f885208a05fa.png">

@mcarrano @mceledonia I need someone who can review it and let me know that the SVG file is good to go so that I can open the PR to update Icon exporter and patternfly-icons.zip

**Comment by mcarrano** [comment]
@doruskova following up on our conversation today, I think we should also add the other risk icons being used from font awesome (i.e., important, moderate, and any others) to our pf-icon set so they show up on the Icons page. Can you identify what those are and we should be able to handle as part of this issue. Those will also need to be added to the official PF icon set as part of the icon exporter sketch file. Thanks.

**Comment by doruskova** [comment]
@mcarrano Here's the list of mentioned icons from Fontawesome :) and updated Icon exporter with a zip file. It includes all icons that we discussed. 

SVGs for 3 other icons  - [SVGs.zip](https://github.com/patternfly/patternfly/files/8350797/SVGs.zip)
Updated Icon exporter - [icon-exporter.sketch.zip](https://github.com/patternfly/patternfly/files/8351535/icon-exporter.sketch.zip)
Updated zip file - [patternfly-icons.zip](https://github.com/patternfly/patternfly/files/8352095/patternfly-icons.zip)
- Important total risk - [Fontawesome link](https://fontawesome.com/v5/icons/angle-double-up?s=solid)
- Moderate total risk - [Fontawesome link](https://fontawesome.com/v5/icons/equals?s=solid)
- Low total risk - [Fontawesome link](https://fontawesome.com/v5/icons/angle-double-down?s=solid)

**Comment by mcarrano** [comment]
Thanks @doruskova . When you return from PTO, can you also send a PR to update the master set in the design repo here? https://github.com/patternfly/patternfly-design/tree/master/resources/patternfly-icon-svgs

@mcoker Let me know if you need any additional input for adding these to the release. And @evwilkin do we need an -org issue to update the Icons page on the website? If so, I can open one.

**Comment by mcoker** [comment]
@mcarrano @mmenestr what should these last 3 icons be named? The name in the comment, or the name of the source FA icon? FYI - I named the first one "critical-risk"

* "important-total-risk" or "angle-double-up"
* "moderate-total-risk" or "equals"
* "low-total-risk" or "angle-double-down"

**Comment by mcoker** [comment]
Actually I just checked the react-icons package and core and we already have the 3 FA icons - angle-double-up, equals, and angle-double-down. Is the goal to duplicate them in the pficons package with the new names?

**Comment by mcarrano** [comment]
@mcoker No, I don't think we need to duplicate them. They weren't showing in the icon package we advertise on the website, but that's a different issue. So from your perspective, it sounds like adding the new "critical-risk" icon is the only thing needed.

**Comment by mcoker** [comment]
@mcarrano yeah I think so. Here are the icons from react-icons and FA in core - https://codesandbox.io/s/admiring-shirley-qj2g0j?file=/fonts.css

The only case where we might want to duplicate them is if we plan on using the critical icon with the others where one icon is swapped out in the other's place and we don't want any sort of shift to happen. Since the critical icon (a pficon icon) and the others (fa icons) come from different icon fonts in core and likely will have small differences in size/shape in the react-icons SVGs, swapping them out with one another may cause layout shifts. That happened when we created the `attention-bell` icon for the notification badge and tried to use it interchangeably with the FA `attention` icon - when we showed one icon in place of the other, the icons shifted a bit since they came from different sources. We ended up duplicating the FA `attention` icon in the pficon set so that the `attention` and `attention-bell` icons both came from the same icon set and were identical in every way other than the addition we made to the bell for the `attention-bell` icon.
